### ISSUE TITLE
Add production-grade open-source Tact projects

### ISSUE THREAD

**Issue by anton-trunov** [tact-lang/awesome-tact#35]

**Comment by anton-trunov** [comment]
@pantemon Are you ok with adding https://github.com/henchtab/nenuma as an example to `awesome-tact`?

**Comment by pantemon** [comment]
I will be pleased to do that! By the way, I am preparing a case study based on the Nenuma Protocol. I am planning to release them by the end of the next week. Hopefully, it will be a nice read for beginners building on top of TON with Tact. Despite the fact that there are lots of way to improve the code; what's we're currently doing.

**Comment by pantemon** [comment]
Do you need anything extra info from me?

**Comment by anton-trunov** [comment]
> Do you need anything extra info from me?

@pantemon yeah, would be nice if you added the link and some short description of your project into the [`smart-contracts-examples`](https://github.com/tact-lang/awesome-tact?tab=readme-ov-file#-smart-contracts-examples) section
